apicast/src/apicast.lua

```diff
@@ -30,6 +31,41 @@ function _M.init()
   end
 end
 
+function _M.init_worker()
+  local interval = tonumber(os.getenv('AUTO_UPDATE_INTERVAL'), 10)
```

Comment:
Right.
Do we need to specify that 10? It's the default value and what everyone would expect. Am I missing something?